yang/frr-bgp-route-map.yang

```diff
@@ -532,18 +532,18 @@ identity set-extcommunity-color {
 
   typedef color-list {
     type string {
-      pattern '((429496729[0-5]|42949672[0-8][0-9]|'
+      pattern '((00|01|10|11):(429496729[0-5]|42949672[0-8][0-9]|'
       +	    '4294967[0-1][0-9]{2}|429496[0-6][0-9]{3}|'
       +	    '42949[0-5][0-9]{4}|4294[0-8][0-9]{5}|'
       +	    '429[0-3][0-9]{6}|42[0-8][0-9]{7}|'
       +	    '4[0-1][0-9]{8}|[1-3][0-9]{9}|'
       +	    '[1-9][0-9]{0,8})(\s*))+';
     }
     description
-      "The color-list type represent a set of colors of value (1..4294967295)
+      "The color-list type represent a set of colors of value (examples 00:200 01:200 10:200)
        values are separated by white spaces";
     reference
-      "RFC 9012 - The BGP Tunnel Encapsulation Attribute";
+      "RFC 9256 - Segment Routing Policy Architecture";
```

Comment:
I suggest keeping both references... Because 9256 clearly does not define the structure explicitly.